docs/guides/src/main/server/config-mainaspects.adoc

```diff
@@ -0,0 +1,128 @@
+<#import "/templates/guide.adoc" as template>
+<#import "/templates/kc.adoc" as kc>
+
+<@template.guide
+title="Core concepts for configuring Keycloak"
+summary="Understand main aspects around Keycloaks configuration">
+
+With the underlying Quarkus distribution, configuring Keycloak got a lot easier. There are some main aspects to understand, though. This guide will help you to understand the core concepts around the configuration of Keycloak.
+
+== Distinction between runtime- and buildtime options
+Configuration options are divided in two categories:
+
+- Runtime Options are options that can be set at runtime when starting the server
```

Comment:
In my proposal, I tried to avoid exposing to users additional complexity by not mentioning what a runtime/build time option is. But just say that some are only available when building.

This is one of the toughest concepts to explain and bringing these "internal" concepts I think does not help to make it easier for users.